admin/public/js/components/host_stats.jsx

```diff
@@ -0,0 +1,45 @@
+define(['pubsub', 'react', 'jsx!components/charts'], function (PubSub, React, MovingLineChart) {
+
+  var xdomain_length = 300;
+  var DATA_INTERVAL = 5000;
+
+  return React.createClass({
+    displayName: 'HostStats',
+
+    getInitialState: function() {
+      return { cpu: 0, mem: 0 };
+    },
+
+    updateHostData: function() {
+      var that = this;
+
+      PubSub.broadcast("ErizoAgent.getErizoAgents", null, function(resp) {
+        that.setState({
```

Comment:
A good place for `if(this.isMounted()) {` (just before this). Basically that's necessary in any async code that calls component methods.